stat_test.go

```diff
@@ -13,6 +13,51 @@ import (
 	"github.com/gonum/floats"
 )
 
+func ExampleCircularMean() {
+	x := []float64{0, 2 * math.Pi, 4 * math.Pi, 6 * math.Pi, 8 * math.Pi}
```

Comment:
Please include an example with numbers that aren't all zero. (maybe 0, pi/3, 2pi/3, pi or something)